diff --git a/pythondata_cpu_vexriscv/verilog/Makefile b/pythondata_cpu_vexriscv/verilog/Makefile
index 7f042ff..2cc9d18 100644
--- a/pythondata_cpu_vexriscv/verilog/Makefile
+++ b/pythondata_cpu_vexriscv/verilog/Makefile
@@ -13,7 +13,7 @@ VexRiscv_Debug.v: $(SRC)
 	sbt compile "runMain vexriscv.GenCoreDefault -d --hardwareBreakpointCount 2 --outputFile VexRiscv_Debug"
 
 VexRiscv_Lite.v: $(SRC)
-	sbt compile "runMain vexriscv.GenCoreDefault --iCacheSize 2048 --dCacheSize 0 --mulDiv true --singleCycleShift false --singleCycleMulDiv false  --outputFile VexRiscv_Lite"
+	sbt compile "runMain vexriscv.GenCoreDefault --iCacheSize 2048 --dCacheSize 0 --mulDiv true --singleCycleShift false --singleCycleMulDiv false	--outputFile VexRiscv_Lite"
 
 VexRiscv_LiteDebug.v: $(SRC)
 	sbt compile "runMain vexriscv.GenCoreDefault -d --iCacheSize 2048 --dCacheSize 0 --mulDiv true --singleCycleShift false --singleCycleMulDiv false --outputFile VexRiscv_LiteDebug"
@@ -27,6 +27,9 @@ VexRiscv_IMAC.v: $(SRC)
 VexRiscv_IMACDebug.v: $(SRC)
 	sbt compile "runMain vexriscv.GenCoreDefault --csrPluginConfig all --atomics true --compressedGen true -d --hardwareBreakpointCount 2 --outputFile VexRiscv_IMACDebug"
 
+VexRiscv_IMACNoDcache.v: $(SRC)
+	sbt compile "runMain vexriscv.GenCoreDefault --iCacheSize 2048 --dCacheSize 0 --csrPluginConfig all --atomics true --compressedGen true --outputFile VexRiscv_IMACNoDcache"
+
 VexRiscv_Min.v: $(SRC)
 	sbt compile "runMain vexriscv.GenCoreDefault --iCacheSize 0 --dCacheSize 0 --mulDiv false --singleCycleShift false --singleCycleMulDiv false --bypass false --prediction none --outputFile VexRiscv_Min"
 
diff --git a/pythondata_cpu_vexriscv/verilog/project/build.properties b/pythondata_cpu_vexriscv/verilog/project/build.properties
index 72f9028..46e43a9 100644
--- a/pythondata_cpu_vexriscv/verilog/project/build.properties
+++ b/pythondata_cpu_vexriscv/verilog/project/build.properties
@@ -1 +1 @@
-sbt.version=1.2.7
+sbt.version=1.8.2
